pinot-core/src/main/java/org/apache/pinot/core/util/ListenerConfigUtil.java

```diff
@@ -167,24 +172,38 @@ private ListenerConfigUtil() {
     return listeners;
   }
 
-  private static ListenerConfig buildListenerConfig(PinotConfiguration config, String namespace, String protocol,
+  private static ListenerConfig buildListenerConfig(PinotConfiguration config, String namespace, String name,
```

Comment:
（minor) why changing the parameter from `protocol` to `name`? `protocol` seems more clear to me

Reply:
the listener config originally required the listener name to be the protocol. now, the protocol can be configured separately from the listener name, i.e. it's possible to have two HTTPS listeners - one internally and the other externally.

Reply:
The idea is that you can run multiple listeners for the same protocol.

For example, running 2 https listeners serving different certificates with different keystore configurations. In that case, protocol is duplicate.

Reply:
I win.